openfisca_france/parameters/prestations/prestations_familiales/aeeh/taux_incapacite_minimal.yml

```
@@ -0,0 +1,6 @@
+taux_incapacite_minimal_aeeh:
+  reference: https://www.legifrance.gouv.fr/codes/section_lc/LEGITEXT000006073189/LEGISCTA000006156691/#LEGIARTI000006750709
+  description: Taux d'incapacité  minimal à partir duquel l'enfant peut prétendre à l'aeeh
+  values:
+    2002-04-24:
+      value: 0.5
```

De même, ajouter un retour à la ligne : 
```suggestion
      value: 0.5

```